detox/src/ios/expectTwo.js

```diff
@@ -105,8 +108,9 @@ class InternalExpect extends Expect {
 }
 
 class Element {
-  constructor(invocationManager, matcher) {
+  constructor(invocationManager, matcher, emitter) {
```

Comment:
I would advise putting emitter before matcher (keeping matcher as the last arg)

Reply:
Why does it matter?

Reply:
NVM, did so